src/main/java/teammates/ui/pagedata/FeedbackSubmissionEditPageData.java

```diff
@@ -20,6 +21,7 @@
 
 public class FeedbackSubmissionEditPageData extends PageData {
     public FeedbackSessionQuestionsBundle bundle;
+    private Map<String, StudentAttributes> studentCourseDetails;
```

Comment:
@tran-tien-dat currently `recipients` inside `recipientList` is a mapping from email to name. Is it okay if I modify it to be a mapping like `email -> StudentAttributes` ? Also this will be incorporated in `FeedbackQuestionsLogic` since it is responsible for constructing the `recipients` hashmap ?

Reply:
just realized a problem. Depending on the  feedback path the value in the hashmap can't be generically changed to `StudentAttributes`. If feedback is being given to teams, instructors etc. it won't work ?

Reply:
@tran-tien-dat so should i go ahead with this approach ?

Reply:
Sorry I missed out these comments and only see them now. No you shouldn't pass a generic `Object` and then use `instanceOf` to determine the behaviour. That is the very last resort and usually implies a bad design.

One way to solve the problem would be to create a class for the `FeedbackSessionQuestionsBundle.recipientList` that can express any type that a recipient can be.